src/ParameterDistributions.jl

```diff
@@ -691,7 +689,7 @@ function logpdf(pd::ParameterDistribution, xarray::AbstractVector{FT}) where {FT
     batches = batch(pd)
 
     # perform the logpdf of each of the distributions, and returns their sum    
-    return sum(cat([logpdf(d, xarray[batches[i]]) for (i, d) in enumerate(pd.distribution)]..., dims = 1))
+    return sum(sum(logpdf(d, xarray[batches[i]])) for (i, d) in enumerate(pd.distribution))
```

Comment:
Here I sum over each `logpdf` evaluation for each batch, then I sum the summed quantities.
The inner sum replaces the need for `cat(..., dims=1)`. Faster and less allocating